cluster-scope/overlays/nerc-ocp-test/nodenetworkconfigurationpolicies/vlan-2173-nese.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: nmstate.io/v1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: vlan-2173-nese
+spec:
+  nodeSelector:
+    node-role.kubernetes.io/worker: ""
+  desiredState:
+    interfaces:
+    - name: bond0.2173
+      type: vlan
+      state: up
+      ipv4:
+        enabled: true
+        dhcp: true
+        auto-routes: true
+      vlan:
+        base-iface: bond0
+        id: 2173
```

Comment:
is it confirmed that this is, in fact, the right VLAN to access the ceph pool for the test cluster?

2177 is for nerc-ocp-infra
2173 is for nerc-ocp-prod

Reply:
@jtriley ^^Could you confirm the VLAN?

Reply:
Should be 2175 for the test cluster

Reply:
Looks like the interface is 2175: [https://massopencloud.slack.com/archives/C027TDE52TZ/p1678481163497359?thread_ts=1676390735.116989&cid=C027TDE52TZ](https://massopencloud.slack.com/archives/C027TDE52TZ/p1678481163497359?thread_ts=1676390735.116989&cid=C027TDE52TZ). Will update when confirmed